packages/fetch/src/lib/types.ts

```diff
@@ -80,3 +80,38 @@ export const enum FetchMethods {
 	 */
 	Patch = 'PATCH'
 }
+
+/**
+ * The list of [Body Media Content Types](https://www.iana.org/assignments/media-types/media-types.xhtml)
+ */
+export const enum FetchBodyMediaContentTypes {
+	/**
+	 * The `JSON` body media content type.
+	 * @see [Media-Types / JSON](https://www.iana.org/assignments/media-types/application/json)
+	 */
+	JSON = 'application/json',
+	/**
+	 * The `XML` body media content type.
+	 * @see [Media-Types / XML](https://www.iana.org/assignments/media-types/application/xml)
+	 */
+	XML = 'application/xml',
+	/**
+	 * The `X-WWW-FORM-URLENCODED` body media content type.
+	 * @see [Media-Types / X-WWW-FORM-URLENCODED](https://www.iana.org/assignments/media-types/application/x-www-form-urlencoded)
+	 */
+	FORM_URLENCODED = 'application/x-www-form-urlencoded',
+	/**
+	 * The `FORM-DATA` body media content type.
+	 * @see [Media-Types / FORM-DATA](https://www.iana.org/assignments/media-types/application/form-data)
+	 */
+	FORM_DATA = 'multipart/form-data',
+	/**
+	 * The `TEXT/HTML` body media content type.
+	 * @see [Media-Types / TEXT/HTML](https://www.iana.org/assignments/media-types/text/html)
+	 */
+	TEXT_HTML = 'text/html',
+	/**
+	 * The `TEXT/PLAIN` body media content type.
+	 */
+	TEXT_PLAIN = 'text/plain'
+}
```

Comment:
```suggestion
/**
 * A list of common [Media Content Types](https://www.iana.org/assignments/media-types/media-types.xhtml) as defined by the IANA.
 */
export const enum FetchMediaContentTypes {
	/**
	 * The `JSON` media content type.
	 * @see [Media-Types / JSON](https://www.iana.org/assignments/media-types/application/json)
	 */
	JSON = 'application/json',
	/**
	 * The `XML` media content type.
	 * @see [Media-Types / XML](https://www.iana.org/assignments/media-types/application/xml)
	 */
	XML = 'application/xml',
	/**
	 * The `X-WWW-FORM-URLENCODED` media content type.
	 * @see [Media-Types / X-WWW-FORM-URLENCODED](https://www.iana.org/assignments/media-types/application/x-www-form-urlencoded)
	 */
	FORM_URLENCODED = 'application/x-www-form-urlencoded',
	/**
	 * The `FORM-DATA` media content type.
	 * @see [Media-Types / FORM-DATA](https://www.iana.org/assignments/media-types/application/form-data)
	 */
	FORM_DATA = 'multipart/form-data',
	/**
	 * The `TEXT/HTML` media content type.
	 * @see [Media-Types / TEXT/HTML](https://www.iana.org/assignments/media-types/text/html)
	 */
	TEXT_HTML = 'text/html',
	/**
	 * The `TEXT/PLAIN` media content type.
	 */
	TEXT_PLAIN = 'text/plain'
}
```

They are just media content types, not Body media content types. Just read the IANA link you linked, it doesn't mention body at all:
> Procedures for registering Media Types can be found in [[RFC6838](https://www.iana.org/go/rfc6838)], [[RFC4289](https://www.iana.org/go/rfc4289)], 
and [[RFC6657](https://www.iana.org/go/rfc6657)]. Additional procedures for registering media types for transfer via 
Real-time Transport Protocol (RTP) can be found in [[RFC4855](https://www.iana.org/go/rfc4855)].

Reply:
Why are all members in SCREAMING_CASE, when we exclusively use PascalCase for enums (and so does djs)?

Reply:
Yeah, I thought the same thing, I wasn't sure what to put